src/Spryker/Zed/CodeGenerator/CodeGeneratorConfig.php

```diff
@@ -104,7 +104,11 @@ public function getProviderNameSpace(GeneratorProjectTypeResolverInterface $type
      */
     public function getProjectType(): string
     {
-        return $this->get(CodeGeneratorConstants::CODE_GENERATOR_PROJECT_TYPE);
+        if (\class_exists('SprykerShop\Yves\ShopApplication\Plugin\Provider\AbstractYvesControllerProvider')) {
```

Comment:
No \ prefixing please.
The class name could be a config value instead of inline. You are using it again in the other code anyway.